### ISSUE TITLE
Make it easier to tell the ingredient column from the product column

### ISSUE THREAD

**Issue by CricketBr** [shpaass/yafc-ce#318]
In this image, the CO2 and storehouse are products, but it's hard to follow the line down from the header. A faint line between columns would work.
![image](https://github.com/user-attachments/assets/62fd4f07-2e03-4ec0-9cdf-7251bd170cdd)

**Comment by shpaass** [comment]
Yup. There was an issue about it somewhere.

**Comment by shpaass** [comment]
Here it is: https://github.com/shpaass/yafc-ce/issues/17
Are you fine with me closing this issue as a duplicate?

**Comment by CricketBr** [comment]
Yes, absolutely fine with that. I should have upvoted that one instead of starting a new one. Thanks!
